src/java/org/jivesoftware/openfire/plugin/util/cluster/ClusterPacketRouter.java

```diff
@@ -40,9 +41,11 @@
     public boolean routePacket(byte[] nodeID, JID receipient, Packet packet) {
         // Send the packet to the specified node and let the remote node deliver the packet to the recipient
         try {
+            if (!ClusterManager.isClusterMember(nodeID))
```

Comment:
This is picky style thing, I know, but can we have braces around the statement for this if() statement please? It may help prevent bugs in the future as this code changes.

Reply:
done